### ISSUE TITLE
s3_only_allow_ssl  KeyError: 'Statement'

### ISSUE THREAD

**Issue by mbassett** [dome9/cloud-bots#150]
Hey guys, when activating the s3_only_allow_ssl bot I'm getting a KeyError: 'Statement'

doing a little digging, I can see that 
policy_bucket = s3_client.get_bucket_policy(Bucket=entity['name'])

returns back a response element, but it does not have a property 'Statement'

'Statement' is instead found  under  'Policy'

when i look at  policy_bucket['Policy']  the results are a string

so  I think maybe this needs to be something like  

policy_bucket = s3_client.get_bucket_policy(Bucket=entity['name'])
policy = json.loads(policy_bucket['Policy'])

In short, this whole script seems to be keying off a field that has been moved.
